test/testfiles/TsxParser_TsFile.ts

```diff
@@ -0,0 +1,17 @@
+import * as React from "react";
+import { Component, type CommonProps } from "components";
+
+interface WrappedComponentProps extends CommonProps {
+    count: number;
+}
+
+export class WrappedComponent extends React.Component<WrappedComponentProps> {
+    render() {
```

Comment:
Need more types in here to verify that it is parsing typescript and not just jsx

Reply:
Added explicit return type to the method